app/src/test/java/com/hedvig/app/ComputedValuesTest.kt

```diff
@@ -0,0 +1,113 @@
+package com.hedvig.app
+
+import com.google.common.truth.Truth.assertThat
+import com.hedvig.app.feature.embark.computedvalues.TemplateExpressionCalculator
+import org.junit.Test
+
+class ComputedValuesTest {
+
+    private val emptyStore = hashMapOf<String, String>()
+
+    @Test
+    fun testSubstitutingANumber() {
```

Comment:
Not necessary for merging, but when doing JDK tests with Kotlin, you can use the following fantastic syntax:

```suggestion
    fun `test substituting a number`() {
```

Reply:
Also, given that this is a function in a test-suite more or less, I think having `test` in the name might be redundant 😄